### PR TITLE
simplebnf:0.1.0

### PR DESCRIPTION
**Pull request by Zeta611**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: simplebnf is a simple package to format Backus-Naur form. The package provides a simple way to format Backus-Naur form (BNF). It provides constructs to denote BNF expressions, possibly with annotations.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [ ] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/simplebnf/0.1.0/simplebnf.typ [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      left,
+      left,
+    ),
+    inset: 2.7pt,
+    stroke: none,
+    ..content,
+  )
+}
+
+#let Prod(
```
Comment (elegaanz) [inline packages/preview/simplebnf/0.1.0/simplebnf.typ:27]: Typst packages generally use kebab-case names, without capital letters. If you want to match this convention, you should rename `Prod` and `Or`. But if there is a good reason to have them capitalized (or if you just prefer it this way), you can keep it as is too.

**Review by elegaanz — COMMENTED** [review]
Thanks for the new package. Everything looks good, just a remark about naming conventions. If you want to keep your function named as they are, just tell me and I'll merge.

**Comment by Zeta611** [comment]
Yes, I am aware of the convention, but renaming Or to or is not possible as it is reserved. So I capitalized them as it matches a common convention of capitalizing constructors in PL.

**Comment by elegaanz** [comment]
Oh right, LGTM then.

**Pull request merged**